mountinfo/mounted_linux_test.go

```diff
@@ -15,252 +15,247 @@ import (
 
 // tMount is a wrapper for unix.Mount which is used to prepare test cases.
 // It skips the test case if mounting is not possible (i.e. user is not root),
-// adds more context to mount error, if any, and installs a cleanup handler to
-// undo the mount.
-func tMount(t *testing.T, src, dst, fstype string, flags uintptr, options string) error {
+// fails the test case if there is a mount error, and installs a cleanup
+// handler to undo the mount.
+func tMount(t *testing.T, src, dst, fstype string, flags uintptr) {
 	if os.Getuid() != 0 {
 		t.Skip("root required for mounting")
 	}
 
-	err := unix.Mount(src, dst, fstype, flags, options)
+	err := unix.Mount(src, dst, fstype, flags, "")
 	if err != nil {
-		return &os.PathError{Path: dst, Op: "mount", Err: err}
+		t.Helper()
```

Comment:
Can you put this one at the start of the function? I think it's the more natural place to put it; also makes sure that the `t.Skip()` shows the right line-number of the test, not the helper.

Reply:
Thanks! Done.